README.md

```diff
@@ -47,8 +47,8 @@ to the default math syntax.
 |                             | cos(0)cos(0) = 1    | cos(0)*cos(0) = 1.0 |
 |                             | 2sin(1) = 1.682942  | 2*sin(1) = 1.682942 |
 |                             | sin(1)2 = 1.682942  | sin(1)*2 = 1.682942 |
-|                             | (2*3)(3*2) = 36     | (2*3)*(3*2) = 36    |
-|                             | 2(3*2) = 12         | 2*(3*2) = 12        |
+|                             | (2\*3)(3\*2) = 36   | (2\*3)*(3\*2) = 36  |
```

Comment:
This line got rendered incorrectly as `| (23)(32) = 36 | (23)(3*2) = 36 |`. Escaping the asterisks seems to fix it.